### ISSUE TITLE
Add a /give "Give" Page to Highlight Donations and Volunteer Opportunities

### ISSUE THREAD

**Issue by allella** [hackgvl/hackgreenville-com#161]
This is an extension of the "sponsor" issue #44, which had a [more recent conversation about donations](https://github.com/hackgvl/hackgreenville-com/issues/44#issuecomment-1634731590).

Though, we don't have any sponsors and we could wait until a RefactorGVL relationship is formalized before getting into sponsorships.

In the meantime, there are some opportunities for folks to give their time or money.

**Donations**

- Build Carolina - Carolina Code School, SC Codes, Develop Carolina
- RefactorGVL
- Synergy Mill

**Volunteer**
- Carolina Codes Conference
- Synergy Mill (gardening, shop clean up)
- Hack For Good (ad hoc projects usually coordinated in [#hack-for-good on the HG Slack](https://hackgreenville.com/join-slack)
- HackGreenville Labs - contributing code through [this GitHub org](https://github.com/hackgvl/) and in [#hg-labs on the HG Slack](https://hackgreenville.com/join-slack)
- HackGreenville Nights
- SC Codes

**Comment by JSn1nj4** [comment]
Is it ok if I jump in on this one?

**Comment by JSn1nj4** [comment]
@ThorntonMatthewD Thanks, I'll take that as a "heck yes" 🙂

**Comment by ThorntonMatthewD** [comment]
Haha, you are spot on!!! Thank YOU!

**Comment by JSn1nj4** [comment]
@allella I have some questions I wanted to ask about this ticket (and I'm sorry for not asking some of this sooner - it probably would've reduced gaps a bit).

## Design

Is it meant for this to be a static page or dynamically load the right orgs based on whether they have Donation and/or Volunteer info? I assumed static when I started, so maybe that's fine at first and it can _become_ dynamic in a separate PR after #210?

The rough idea I had was:
- 2 sections each with a max 3-column layout (Or 1 or 2 centered when not enough on a row using flexbox)
- Name, blurb, "learn more" link to the relevant page

![image](https://github.com/hackgvl/hackgreenville-com/assets/5084820/52d65062-cf43-469a-9881-5d6e61767013)

If not this, it can be something like:
- 1 row per org
- name/logo on left, blurb/longer description + "Learn More" button on right

## Orgs

Is this still correct?

>Donations
> - [Develop Carolina][dev_carolina]
> - [RefactorGVL][ref_gvl]
> - [Synergy Mill][syn_donate]
>
> Volunteer
> - [Carolina Code Conference][ccc]
> - [Synergy Mill][syn] (gardening, shop clean up)
> - ~~Hack For Good~~ [Code with the Carolinas][code_carolinas_volunteer] (ad hoc projects usually coordinated in [#hack-for-good on the HG Slack](https://hackgreenville.com/join-slack)
> - [HackGreenville Labs][hg_labs] - contributing code through [this GitHub org](https://github.com/hackgvl/) and in [#hg-labs on the HG Slack](https://hackgreenville.com/join-slack)
> - [HackGreenville Nights][hg_nights]

[ccc]: https://carolina.codes
[code_carolinas_volunteer]: https://codewiththecarolinas.org/volunteer.html
[dev_carolina]: https://www.developcarolina.org/
[hg_labs]: https://hackgreenville.com/labs
[hg_nights]: https://hackgreenville.com/hg-nights
[ref_gvl]: https://refactorgvl.com/
[syn]: https://www.synergymill.com/
[syn_donate]: https://www.synergymill.com/donate
[syn_fb]: https://www.facebook.com/synergymill

Also trying to make sure all the links make sense. For instance:
- Develop Carolina doesn't _currently_ have donation page.
- RefactorGVL has a donate link, but it's straight to a Stripe form. Should we link to the RefactorGVL site first in case people want to see who they're donating to first?
- Synergy Mill has a [donate][syn_donate] page, but I don't see a volunteer page. For volunteering, is it fine to link to the homepage or should we send visitors somewhere like the [Facebook page][syn_fb]?

**Comment by ThorntonMatthewD** [comment]
I personally really like the 3>= column layout - the prototype looks fantastic!

For Develop Carolina:
 I know that their parent organization, Build Carolina, has a [page](https://buildcarolina.org/build-carolina-giving) to accept monetary donations. Leila King, the director, is in the HG Slack and could confirm further.

**Comment by JSn1nj4** [comment]
Oh awesome, thanks! I'll reach out to her and see what she says.

**Comment by allella** [comment]
I would go with static on anything like this while we're figuring out what we're doing.  Get it done "on paper" via HTML first and unless we need views or filtering or individual pages, or some future features, then it probably doesn't need to be in a database.

Yeah, Build Carolina could be for the donation and perhaps list their imitative names under that donate (Carolina Code School, SC Codes, Develop Carolina, they have a new MVP initiative that may not be named).

They have volunteer ops with SC Codes as mentors.

**Comment by allella** [comment]
I fixed the list above.  We'll likely be adding more at some point, so having a flexible list like that is good.

A link to Synergy Mill for volunteering is good.  People might need to join their Slack to find out, so we can see if Synergy Mill has input once we have things live.

Agreed to send RefactorGVL's to the homepage instead of the Stripe.  We could note that HackGreenville is an initiative and donations could be earmarked to HG via RefactorGVL.

[Build Carolina's donate page](https://buildcarolina.org/build-carolina-giving)

**Comment by allella** [comment]
@raiderrobert we're talking about linking to Refactor's homepage for giving.  

I don't imagine a lot of money flowing in at this point, but is it easy to add an "earmark" field to the donate field to see if anybody wants to direct a donation to HG funds?

**Comment by raiderrobert** [comment]
@allella I made a field on the donation page where a user can requested an earmark. 

Does that work for you all?

<img width="1048" alt="Screenshot 2024-01-26 at 3 55 15 PM" src="https://github.com/hackgvl/hackgreenville-com/assets/806571/aeee7d70-f2e9-46fa-8746-14853813a5d6">

**Comment by allella** [comment]
Thanks. Good enough for now.
